contributing_to_docs/contributing.adoc

```diff
@@ -8,10 +8,14 @@
 
 toc::[]
 
+[NOTE]
+====
+Are you contributing content for 4.0? If yes, please read how to xref:contributing_user_stories.adoc[contribute user stories].
+====
```

Comment:
The link to contributing user stories does not work for me. Anyone else experience this?

Reply:
@tmorriso-rh - Hey Traci - where are you testing this link? This links to the new file that is a part of this PR.

Reply:
LOL - I found the error after I had merged it. I forgot this shouldn't be an AsciiDoc link.